web/cobrands/sass/_layout.scss

```diff
@@ -272,6 +273,7 @@ body.mappage {
       left: 0;
       right: 0;
       background: $map_nav_bg;
+      height: $mappage-header-height;
```

Comment:
`$mappage-header-height` should really only apply to headers on map pages (otherwise we should change the variable name!) – so I've changed the `height` here to `min-height`, which will allow the nav to break onto multiple lines (https://github.com/mysociety/fixmystreet/pull/1406/commits/3850ec485da3ea27a18013f6d80c68bca04279b9#diff-d1a25ba8f0058d138eb8b66a0a772bd8R280)